### ISSUE TITLE
New repository for the JSON API

### ISSUE THREAD

**Issue by ku-antum** [IBM/qiskit-api-py#7]
As quantum-experience has a JSON API that is independent of its python implementation could you just add another repository for the definition of the API alone (w/o any implementation)?
It would also be useful for collecting JSON API change requests and improvement suggestions.

(I usually use my homemade Java client for running experiments and so far have to read through the python implementation to find out about the different API requests, parameters, and possible responses.)

Best regards,
kuantum

**Comment by pacomf** [comment]
Yes @ku-antum this is the idea. We are improving and documenting our REST API to release all methods and every body can use it.

Thank you very much for your proposal! We are proud that the community helps to improve QX. :), if your have any other idea to improve QX, don't hesitate to contact us.
Paco.

**Comment by ku-antum** [comment]
Thanks, @pacomf
That's great news. Is a first release date known?
Or, could you put a simple pre-alpha description in the repo here?

**Comment by pacomf** [comment]
We are working hard, we hope ready the public REST API in few weeks :),

**Comment by ku-antum** [comment]
Oh, that was a very quick reply :-)
Would a pre-alpha test release be possible sooner?
That could help with testing, too.

**Comment by madhatter197627** [comment]
Did it get reviews

**Comment by MarcLapierre** [comment]
👋 Has the REST API documentation been released / is it possible to have a look at an in-progress version?
